src/routes/ConsortiumManager/settings/users/general/PermissionSets/PermissionSets.js

```diff
@@ -125,6 +134,8 @@ export const PermissionSets = (props) => {
     >
       <Switch>
         <Route exact path={PERMISSION_SET_ROUTES.COMPARE} component={PermissionSetsCompare} />
+        <Route exact path={PERMISSION_SET_ROUTES.CREATE} component={PermissionSetsCreate} />
+        <Route exact path={`${PERMISSION_SET_ROUTES.EDIT}/:id`} component={PermissionSetsCreate} />
```

Comment:
PermissionSetsCreate is not valid for edit, please create `ConsortiumPermissionsSetForm` component that will have all common logic for create/edit and 2 components for create and edit that define how permission set is saved

you'll have
1. `ConsortiumPermissionsSetForm` - almost the same as your current `PermissionSetsCreate`, but you need to make it configurable via props and more generic
2. `PermissionSetsCreate` - for create
3. `PermissionSetsEdit` - for edit